jobs/sql/incremental/previousWatermarkQuery.sql

```diff
@@ -0,0 +1 @@
+select max(last_updated) from ${tableName} where partition_num = ${partitionNum}
```

Comment:
The previous watermark is defined as the max(last_updated) from the target table/partition.  In order for this query to be reused consistenly, we need any target tables that want to do incremental loading ot have a "last_updated" column (we could rename this as long as we do it across the board.